File: webapp/HIN_WEB/Models/Category.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HIN_WEB.Models
{
    public class Category
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string DisplayName { get; set; }
        public bool? IsActive { get; set; }
        public int? EntityTypeId { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public string UpdatedBy { get; set; }
        public int? UpdatedById { get; set; }
        public DateTime? CreatedOn { get; set; }
        public string CreatedBy { get; set; }
        public int? CreatedById { get; set; }
    }
}

File: webapp/HIN_WEB/ClientApp/src/app/model/Category.ts
export class Category {
  public Id: number;
  public Name: string;
  public DisplayName: string;
  public IsActive: boolean;
   public UpdatedOn: Date ;
 public UpdatedBy: string ;
 public UpdatedById: number ;
 public CreatedOn: Date ;
 public CreatedBy: string ;
 public CreatedById: number ;
  public EntityTypeId: number;
}
export class CategoryValues {
  public Id: number;
  public CategoryId: number;
  public EntityId: number;
  public EntityTypeId: number;
  public UpdatedOn: Date;
  public UpdatedBy: string;
  public UpdatedById: number;
  public CreatedOn: Date;
  public CreatedBy: string;
  public CreatedById: number;
}
